tests/tblgen_to_py/test.py

```diff
@@ -50,13 +50,7 @@ class Test_AttributesOp(IRDLOperation):
     name = "test.attributes"
 
     int_attr = prop_def(
-        ParamAttrConstraint(
-            IntegerAttr,
-            (
-                AnyAttr(),
-                EqAttrConstraint(IntegerType(16)),
-            ),
-        )
+        IntegerAttr[IntegerType].constr(type=EqAttrConstraint(IntegerType(16)))
```

Comment:
This file was automatically generated and probably shouldn't be manually changed